build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
I am not sure if we would want to use Kotlin DSL for Gradle. Although the anchor platform uses Kotlin DSL, most public Gradle examples are written in Groovy.

Reply:
Sure, we can use Groovy for Gradle. I saw Kotlin option and just went all in on Kotlin. :sweat: